spec/controllers/annotations_controller_spec.rb

```diff
@@ -365,6 +394,25 @@
         end
       end
 
+      it 'returns a 403 ("forbidden") when group_id does not match' do
```

Comment:
This could likely be a shared example (`it_behaves_like 'uses JWT authorization` or something like that) since we use basically the same tests three times. You can just add the shared example to this file for now - eventually it might be worth extracting into a separate file but no reason to complicate things right now.

Reply:
You might need to `yield` to account for the different actual requests, so maybe the duplication is fine for now since it won't be a straightfoward shared example.

Reply:
Using it_behaves_like would be the best, but I ran into issues passing the actual method call like `post_to_create(@annotation)` and having it execute in the order needed.

Reply:
You should be able to [set the context](https://relishapp.com/rspec/rspec-core/docs/example-groups/shared-examples#providing-context-to-a-shared-group-using-a-block) each time you call the shared example and define a proc to be called to trigger the request. For this one it might be something like:

```ruby
it_behaves_like 'foo' do
  let(:request_proc) do
    ->(anno) { post_to_create(anno) }
  end
end
```

where there is a `request_proc.call(params)` line inside of the test. Again, this isn't super critical so I think we're better off merging this in as-is and opening an issue to refactor these tests down the line.